docs/cuda-miner.md

```diff
@@ -56,13 +56,13 @@ Compilation of CUDA miner is done by invoking:
 
 ```bash
 cd apps/aecuckoo && make c_src/.git
-cd c_src/src && make libblake2b.so cuda28
+cd c_src/src && make libblake2b.so cuda30
```

Comment:
Maybe tag `tags/v0.17.0` above shall be updated to `tags/v0.19.0` because of new cuckoo commit?

Reply:
@lucafavatella I don't find it relevant to this PR/Commits. I can do it in separate one or in the release prep. as always.

Maybe we should start maintaining `tags/latest` ?

Reply:
> the release prep.

I will. Thanks.